controllers/apply/index.js

```diff
@@ -71,6 +73,66 @@ router.get('/emails/unsubscribe', async function(req, res) {
     }
 });
 
+// Cleanup applications with outdated field names
+router.post('/migrate-data', async (req, res) => {
+    if (req.body.secret !== EMAIL_EXPIRY_SECRET) {
+        return res.status(403).json({ error: 'Invalid secret' });
+    }
+
+    const replacements = [
+        {
+            from: 'east-dumbartonshire',
+            to: 'east-dunbartonshire'
+        },
+        {
+            from: 'orkney',
+            to: 'orkney-islands'
+        },
+        {
+            from: 'shetland',
+            to: 'shetland-islands'
+        },
+        {
+            from: 'west-dumbartonshire',
+            to: 'west-dunbartonshire'
+        },
+        {
+            from: 'highlands',
+            to: 'highland'
+        }
+    ];
+
+    const invalidRegionNames = replacements.map(_ => _.from);
+
+    const applications = await PendingApplication.findAllByForm(
+        'awards-for-all'
+    );
+
+    const appsToUpdate = applications.filter(app => {
+        return includes(
```

Comment:
FYI `Array.includes` is supported in the version of node we run so we don't need lodash/includes. Doesn't matter here though.

Reply:
Aha!

Reply:
![image](https://user-images.githubusercontent.com/123386/66477364-d8cc4780-ea8f-11e9-8d0b-c4cce14cb357.png)